src/sorted.js

```diff
@@ -4,6 +4,16 @@ Sk.builtin.sorted = function sorted (iterable, cmp, key, reverse) {
     var iter;
     var compare_func;
     var list;
+    var rev;
+
+    if (reverse === undefined) {
+        rev = false;
+    } else if (reverse === Sk.builtin.none.none$) {
```

Comment:
This check is not sufficient as we can still pass a float etc. I think we need to check if the passed argument is an int/index. Bool is a subtype of int.